### ISSUE TITLE
Screen Reader does not read out MCP server Run status from codelens.

### ISSUE THREAD

**Issue by bhavyaus** [microsoft/vscode#247661]
Testing #247533

	1.	Add an MCP server using the MCP: Add Server... command.
	2.	Verify that the CodeLens indicates the server is Running.
🐛 Issue: The Narrator does not announce the server status (“Running”) after adding the tool.

**Comment by connor4312** [comment]
https://github.com/microsoft/vscode/issues/171755
